solutions/devsprint-matheus-reis-2/DeliveryAppChallenge/Screens/AddressSearch/AddressSearchViewModel.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  AddressSearchViewModel.swift
+//  DeliveryAppChallenge
+//
+//  Created by Bruno Vieira Souza on 18/05/22.
+//
+
+import Foundation
+
+class AddressSearchViewModel {
+    
+    private var service: DeliveryApi
+    
+    private var enderecos: [Address] = []
```

Comment:
Já que todo o app está sendo escrito em inglês, seria mais interessante nomear esse atributo como `addresses`.

```suggestion
    private var addresses: [Address] = []
```

Reply:
corrigido